docs/content/faq.md

```diff
@@ -87,3 +87,27 @@ Which currently includes:
 - hg
 - bzr
 - fossil
+
+### When should I use a vendor directory, and when should I use Athens?
+
+The Go community has used vendor directories for a long time before module proxies like Athens came along, so naturally each group collaborating on code should decide for themselves whether they want to use a vendor directory, use Athens, or do both!
+
+Vendor (without a proxy) are valuable when:
+
+- CI/CD systems don't have access to an Athens (even if it's internal)
+- When the vendor directory is so small that it is still faster to check it out from a repo than it is to pull zip files from the server
+- If you're coming from glide/dep or another dependency management system that leveraged the vendor directory
+
+Athens valuable when:
+
+- You have a new project
+- Your team requires that you use Athens (i.e. for isolation or dependency auditing)
+- Your vendor directory is large and causing slow checkouts and downloading from Athens speeds the build up
+  - For developers slow checkouts will not be as much of a problem as for ci tools which frequently need to checkout fresh copies of the project
+- You do not want to pollute your project with the vendor directory
+  - Extra noise in pull requests
+  - Extra difficulty doing fuzzy file searching in your project
```

Comment:
Can you elaborate on this and the previous bullet point? It sounds like Athens will be valuable in reducing these two things. Is that right?

Reply:
This is awkward phrasing. Is this easier to follow?
```
- You want to remove the vendor directory from your project to:
  - Reduce noise in pull requests
  - Reduce difficulty doing fuzzy file searching in your project
```

Reply:
Most definitely, that's easier to follow!